### DISCUSSION TITLE
lotus v16 update.. windowpost  always fails

### DISCUSSION THREAD

**Discussion by JshFeel** [filecoin-project/lotus#8995] [Build and Sync] [answered]
hi.
I updated v16 and all sectors failed.
Currently, it comes out as lotus version 1.16.0, but the windowpost fails on the second day and the loss continues.
I don't know what more to do to get it back to normal. it's too hard. 
Looking at the release notes, I don't know what the problem is.
Do you have a good friend who can tell you again in detail how to update? i want help too help me.

All sectors being sealed are in ret-wait state.

windowpost keeps failing.

Sectors that have already been probed are not even recovered.
Newly working sectors are ret-wait
![burn](https://user-images.githubusercontent.com/36658732/177902173-25c87b6e-9811-4770-88a6-2942d53e8559.jpeg)
.

**Comment by LexLuthr** [comment, marked as answer]
Hi @JshFeel, I don't see any description of what the problem is except a vague description that WDpost is failing. In the intesrest of quick debugging, I would ask you to update logs and any information you think would helps.

I would also like you to check the below slack conversation and make sure it is not a mpool congestion.
https://filecoinproject.slack.com/archives/C01AC6999KQ/p1657147985393639

You should also checkout https://filecoinproject.slack.com/archives/C01AC6999KQ/p1657220990452119

and https://filecoinproject.slack.com/archives/C01AC6999KQ/p1657231318196299
